readme.md

```diff
@@ -1,7 +1,5 @@
 # Utilshare
 
-![Build Status](https://travis-ci.org/Devlus/util-share.svg?branch=master)](https://travis-ci.org/Devlus/util-share)
```

Comment:
IDK why this keep happening. Something about the capitalization of the readme file is messing up my git history